src/test/java/com/frameworkium/integration/ai/capture/api/dto/sut/Versions.java

```diff
@@ -0,0 +1,28 @@
+package com.frameworkium.integration.ai.capture.api.dto.sut;
+
+
+import com.frameworkium.integration.ai.capture.api.dto.executions.BaseDTO;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Versions extends BaseDTO <Versions> {
+
+    public List<String> versions;
+
+    @SuppressWarnings("WeakerAccess")
+    public static List<String> preExistingFwiumCoreVersions = Arrays.asList("master", "selenium-3.0", "2.3.0",
```

Comment:
If we re-set the Capture instance (as we might in a CI scenario) then hard-coding this data is probably ill-advised. The test should probably set up the data and you should assert that is contains what you put it (even if it may already exist), rather than presuming the system will be in a certain state. This just makes the tests require less maintenance when data changes.